src/components/BooksContainer.js

```diff
@@ -1,12 +1,27 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Books from './Books';
+import { fetchBook } from '../redux/books/booksSlice';
 
 const BooksContainer = () => {
   const bookList = useSelector((state) => state.books.value);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchBook());
+  }, [dispatch]);
+
+  // const fetchBooks = async () => {
+  //   const api = await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/', { method: 'POST' });
+  //   const result = await api.text();
+  // };
+  // useEffect(() => {
+  //   fetchBooks();
+  // }, []);
```

Comment:
- [ ] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

Reply:
Hello @Owusu-Desmond , I don't have any compile error when run it to my local pc. Can you please help me